manifests/init.pp

```diff
@@ -43,6 +43,9 @@
 # @param nat
 #   Add default tables and chains to process NAT traffic.
 #
+# @param nat_table_name
+#   The name of the 'nat' table: defaults to 'nat'
```

Comment:
There's no need to describe here the default value, it will be automatically described in `REFERENCE.md` after regenerating the doc strings.